.env

```diff
@@ -1,7 +1,2 @@
-VEGA_SIM_VEGA_TAG=a7ce1fa22b1bd301753367e4798141c6ffc7ecb0
-VEGA_SIM_CONSOLE_TAG=develop
-VEGA_DEFAULT_KEY_NAME="Key 1"
-VEGA_SIM_NETWORKS_INTERNAL_TAG=main
-VEGA_SIM_NETWORKS_TAG=master
-VEGA_USER_WALLET_NAME="NETWORK_TRADERS"
-VEGA_USER_WALLET_PASS="PASSPHRASE"
```

Comment:
Nor sure all of these were supposed to be removed?

Reply:
Agh, looks like that update script is going to need to be made smarter..